lib/Minz/Extension.php

```diff
@@ -253,16 +255,16 @@ private function isExtensionConfigured(string $type): bool {
 	 * @return array<string,mixed>
 	 */
 	private function getConfiguration(string $type): array {
-		if (!$this->isConfigurationEnabled($type)) {
+		if (!$this->isConfigurationEnabled($type) || !$this->isExtensionConfigured($type)) {
 			return [];
 		}
 
-		if (!$this->isExtensionConfigured($type)) {
-			return [];
+		if ($type === 'user') {
+			return FreshRSS_Context::userConf()->{$this->config_key}[$this->getName()];
+		} elseif ($type === 'system') {
+			return FreshRSS_Context::systemConf()->{$this->config_key}[$this->getName()];
 		}
-
-		$conf = "{$type}_conf";
-		return FreshRSS_Context::$$conf->{$this->config_key}[$this->getName()];
```

Comment:
Also removed the last variable variables `$$` instances, which were blocking stricter levels